lib/upgrade_format_task.rb

```diff
@@ -52,12 +58,12 @@ def upgrade(store, name, options = {})
 
     if options[:all]
       if migrations_done > 0
-        Machinery::Ui.puts "Upgraded #{migrations_done} system descriptions successfully."
+        Machinery::Ui.puts(
+          "Upgraded #{migrations_done} #{Machinery::pluralize("system description", migrations_done)}."
```

Comment:
Line is too long. [103/100]<br>Prefer single-quoted strings inside interpolations.